hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/WritableUpgradeFileStore.java

```diff
@@ -50,24 +52,32 @@ public class WritableUpgradeFileStore extends ReadableUpgradeFileStoreImpl {
      */
     public WritableUpgradeFileStore(@NonNull final WritableStates states) {
         super(states);
-        writableUpgradeState = requireNonNull(states.getQueue(getStateKey()));
+        this.states = requireNonNull(states);
         writableUpgradeFileState = requireNonNull(states.get(BLOBS_KEY));
     }
 
-    public void add(@NonNull final File file) {
+    public void add(@NonNull final File file, @NonNull final FileID fileID) {
```

Comment:
The fileID parameter is no longer needed or used in this method.

Reply:
removed thanks